### PR TITLE
Revise wording to clarify bounding box

### PR DESCRIPTION
**Pull request by PeterConstable**
Addressing #209.

### CODE REVIEW

**Comment by PeterConstable** [comment]
@justvanrossum: your review would be welcomed

**Comment by justvanrossum** [comment]
Looks good to me. But please also consider rephrasing line 350 of colr-gradients-spec.md:
- "Allocate a bitmap for the glyph according to glyf table entry extents for gid"

File: colr-gradients-spec.md [hunk 1]
```diff
@@ -340,7 +340,7 @@ font formats, including COLR v1.
 ### Pseudocode
 
 ```
-Allocate a bitmap for the glyph according to glyf table entry extents for gid
+Allocate a bitmap for the glyph according to extents of glyph contours for gid
```
Comment (justvanrossum) [inline colr-gradients-spec.md:343]: perhaps "... extents of base glyph contours for gid"?

**Comment by justvanrossum** [comment]
Looks good to me, thanks!

**Pull request merged**